src/components/ListItem.vue

```diff
@@ -127,30 +133,55 @@
           index
         })
       },
-      isShow: function (isTrue) {
-        switch (this.view) {
-          case 'all': return true
-          case 'done': {
-            if (isTrue) return true
-            else return false
+      isShow: function (isTrue, id) {
+        if (this.userId === undefined) {
+          switch (this.view) {
+            case 'all': return true
+            case 'done': {
+              if (isTrue) return true
+              else return false
+            }
+            case 'uncompleted': {
+              if (isTrue) return false
+              else return true
+            }
+            default: return true
           }
-          case 'uncompleted': {
-            if (isTrue) return false
-            else return true
+        } else {
+          if (id === this.userId) {
+            switch (this.view) {
+              case 'all': return true
```

Comment:
Should not combine return and switch. You can use a variable with switch, then return based on variable value